openid-caep-specification-1_0.txt

```diff
@@ -630,55 +630,64 @@ Cappalli & Tulshibagwale     Standards Track                   [Page 11]
 
 3.4.1.  Event-Specific Claims
 
-   current_level  REQUIRED, JSON string: the current NIST Authenticator
-      Assurance Level (AAL) as defined in [SP800-63R3]
+   namespace:  REQUIRED, JSON string: the namespace of the values in the
```

Comment:
JSON is technically case sensitive. Should we say that this string is case-insensitive?

Reply:
It might be good to add it as a top-level guidance for anything defined in the specification